src/modules/refresh.mjs

```diff
@@ -0,0 +1,11 @@
+import buildScore from './buildScore.mjs';
+
+const freshBtn = document.querySelector('.refresh');
```

Comment:
- [ ] Kindly ensure that the score board changes only when the refresh button is clicked. it currently changes after clicking the submit button.

https://www.loom.com/share/5e2858b29d304a3f8b66732314ed7deb